### DISCUSSION TITLE
Alert screen with status code 500

### DISCUSSION THREAD

**Discussion by tacitochaves** [Security-Onion-Solutions/securityonion#5841] [General]
Esteemed,

I installed Security Onion, configured port mirroring on the switch, but when I go to alert the following screen and message appear: Requeste falied with status code 500.
![SO - Status 500](https://user-images.githubusercontent.com/6299301/136801160-3bb38629-5553-45f1-ae4e-6cc68eda8905.png)

can anybody help me?

**Comment by dougburks** [comment]
From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:

Please include the following details where you can:

- Security Onion version as seen in the lower left corner of SOC and in /etc/soversion. For example: 2.3.70
- Install source. Did you install from our Security Onion ISO image or did you perform a network installation?
- If network install, did you install from CentOS 7 or Ubuntu 18.04?
- Install type. ex. eval, standalone, forward node, search node, etc
- Does so-status show all services running?
- Do you get any failures when you run salt-call state.highstate from the node?
- Does the grid status show any failures?

**Reply by Rourou4** [reply]
Hi , so i have the same problem as him and when i do so-status so-elasticsearch and so-elastalert not running and i dont know what's the problem can you help me ?

**Reply by dougburks** [reply]
@Rourou4 From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:

**Start a new discussion instead of replying to somebody else's discussion**
_Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion._
